packages/markdown/remark/src/highlight.ts

```diff
@@ -61,8 +67,7 @@ export async function highlightCodeBlocks(tree: Root, highlighter: Highlighter)
 			}
 		}
 
-		// Don’t mighlight math code blocks.
-		if (languageMatch?.[1] === 'math') {
+		if (excludeLanguages.includes(languageMatch?.[1] ?? '')) {
```

Comment:
The code will hit a false positive if a user provides a value such as `excludeLanguages: ['']`. Can we make the check a bit stronger?